networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -112,6 +147,80 @@ TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraphEdgeDeletion) {
 
 }
 
+TEST_P(DynBetweennessGTest, testDynApproxBetweenessGeneratedGraph) {
+    DorogovtsevMendesGenerator generator(200);
+    Graph G1 = generator.generate();
+    Graph G(G1, isWeighted(), isDirected());
```

Comment:
`DorogovtsevMendesGenerator` generates undirected unweighted graphs. If you need to generate a directed graph, you need to use another generator (e.g., `ErdosRenyiGenerator`).